Mathlib/AlgebraicTopology/FundamentalGroupoid/Product.lean

```diff
@@ -55,17 +55,18 @@ theorem proj_map (i : I) (x₀ x₁ : πₓ (TopCat.of (∀ i, X i))) (p : x₀
 #align fundamental_groupoid_functor.proj_map FundamentalGroupoidFunctor.proj_map
 
 -- Porting note: losing the instance with a concrete category again
-instance : (i : I) → TopologicalSpace (πₓ (X i)).α := fun i => TopCat.topologicalSpace_coe (X i)
+--instance : (i : I) → TopologicalSpace (πₓ (X i)).α :=
```

Comment:
Should this be deleted? Or do we need TO DO here?

Reply:
Yeah I think it should be. The fundamental groupoid shouldn't get a topological structure, just like the topological space shouldn't get a category structure. Previously they got mixed up due to `reducible`.